internal/profile/_static/kibana_config_default.yml

```diff
@@ -17,5 +17,6 @@ xpack.fleet.registryUrl: "https://package-registry:8080"
 xpack.fleet.agents.enabled: true
 xpack.fleet.agents.elasticsearch.host: "https://elasticsearch:9200"
 xpack.fleet.agents.fleet_server.hosts: ["https://fleet-server:8220"]
+xpack.fleet.enableExperimental: ["experimentalDataStreamSettings"] # Enable experimental toggles in Fleet UI
```

Comment:
This should be probably added to `kibana_config_8x.yml` only. `kibana_config_80.yml` is used for 8.0 version, and `kibana_config_default.yml` is used for older versions.